tests/functional/tree.t

```diff
@@ -49,10 +49,21 @@ Since the following custom arguments are incompatible with the default IQ-TREE a
   $ ${AUGUR} tree \
   >  --method iqtree \
   >  --alignment tree/full_aligned.fasta \
-  >  --tree-builder-args="-czb -bb 1000 -bnni" \
+  >  --tree-builder-args="--polytomy -bb 1000 -bnni --redo" \
```

Comment:
Is the addition of `--redo` necessary here because we're writing the output of all these tests to `$TMP`? If so, we could just remove references to `$TMP` throughout this file (following [our own cram style guide](https://github.com/nextstrain/augur/blob/master/docs/contribute/DEV_DOCS.md#3-functional-tests)) and skip the `--redo`.

Reply:
I'm not quite sure anymore why I needed to add the `--redo` - there was definitely some issue I hit without it - possibly iqtree stubmling upon an existing file from an earlier test without having cleaned up. 

Redo does no harm (it should be the default, and one should actively opt into checkpoints). It might not bring many benefits but given there's no harm I thought I'd sprinkle it.

Reply:
This would require reorganizing the cram files: https://github.com/nextstrain/augur/issues/1558

I plan to make a PR for that later, but this shouldn't block on it!

Reply:
Actually I'm not sure we can avoid `--redo`. I realized while working on #1559 that IQ-TREE puts intermediate files in the directory of the `--alignment` input file, not the directory of the `--output` file. Only the 2nd is cleaned up after every test.

Reply:
Good discussion here, I've got a fix for this issue in #1667 - because IQtree pollutes the _input_ file directory, it's best to copy the input file data dir to each temporary directory.

We can remove the `-redo` once/if that's merged. For now, I'd prefer to keep it like this to not break something else on the way.